lib/Doctrine/DBAL/Platforms/SqlitePlatform.php

```diff
@@ -570,6 +578,14 @@ public function getForUpdateSql()
     /**
      * {@inheritDoc}
      */
+    public function getInlineColumnCommentSQL($comment)
+    {
+        return "--{$comment}\n";
```

Comment:
Shouldn't we make sure here that `$comment` does not contain `\n`? Otherwise the generated SQL will be invalid. We could either strip newlines or throw exception...